plugin/trino-kafka/src/test/java/io/trino/plugin/kafka/TestKafkaSecurityConfig.java

```diff
@@ -0,0 +1,105 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.trino.plugin.kafka;
+
+import com.google.common.collect.ImmutableMap;
+import io.trino.testing.assertions.Assert;
+import org.testng.annotations.Test;
+
+import java.util.Map;
+
+import static io.airlift.configuration.testing.ConfigAssertions.assertFullMapping;
+import static io.airlift.configuration.testing.ConfigAssertions.assertRecordedDefaults;
+import static io.airlift.configuration.testing.ConfigAssertions.recordDefaults;
+import static org.assertj.core.api.Assertions.assertThat;
+
+public class TestKafkaSecurityConfig
+{
+    @Test
+    public void testDefaults()
+    {
+        // No defaults, should be empty if nothing is set specifically
+        assertRecordedDefaults(recordDefaults(KafkaSecurityConfig.class)
+                .setSslKeystoreFile(null)
+                .setSslKeystorePassword(null)
+                .setSslKeyPassword(null)
+                .setSslTruststoreFile(null)
+                .setSslTruststorePassword(null)
+                .setSslEndpointIdentificationAlgorithm(null));
+    }
+
+    @Test
+    public void testExplicitPropertyMappings()
+    {
+        String keystoreFilepath = getClass().getClassLoader().getResource("keystore.jks").getPath();
```

Comment:
use `pom.xml` instead but please extract it as variable named `dummyExistingFile`

Reply:
What we typically do is:
```
Path keyststore = Files.createTempFile(null, null);
```

Reply:
Hi @losipiuk , I removed those 2 `*.jks` files and added this block instead:

```
        String secret = "confluent";

        Path keysstorePath = Files.createTempFile("keystore", ".jks");
        FileWriter writer = new FileWriter(keysstorePath.toFile(), StandardCharsets.UTF_8);
        writer.write(secret);
        writer.close();
        Path truststorePath = Files.createTempFile("truststore", ".jks");
        writer = new FileWriter(truststorePath.toFile(), StandardCharsets.UTF_8);
        writer.write(secret);
        writer.close();
```